manim/animation/composition.py

```diff
@@ -46,7 +46,7 @@ def __init__(
         super().__init__(self.group, rate_func=rate_func, lag_ratio=lag_ratio, **kwargs)
         self.run_time: float = self.init_run_time(run_time)
 
-    def get_all_mobjects(self) -> Group:
+    def get_all_mobjects(self) -> Sequence[Mobject]:
```

Comment:
hmm, it's an incompatible type, right?

Reply:
is it? You can Iterate it and use group[index] - isn't that all a Sequence needs?

Reply:
Anyway, now it should be fitting :)